packages/legend-application-studio/src/stores/DSL_Mapping_LegendStudioApplicationPlugin_Extension.ts

```diff
@@ -86,9 +87,16 @@ export type RuntimeConnectionTooltipTextBuilder = (
 
 export type NewConnectionDriverCreator = (
   editorStore: EditorStore,
-  store: Store,
+  typeOrStore: Store | string,
```

Comment:
Can we rename this to something else?

Reply:
what would you suggest, not sure I understand the issue with this